scripts/enforceVersions.sh

```diff
@@ -12,15 +12,15 @@ if [[ $(pwd) =~ .+psammead ]]; then
   fi
 
   # cut is used to transform 'version=6.4.1' into '6.4.1' for easy comparison
-  desired_npm_version=$(cat .npmrc | cut -d "=" -f2)
+  desired__version=$(cat .yarnrc | cut -d "=" -f2)
 
-  if [[ $(npm -v) != $desired_npm_version ]]; then
+  if [[ $(yarn -v) != $desired_yarn_version ]]; then
     tput setaf 1
-    echo "\nOh no! You're not using our version of npm, please move to version $desired_npm_version in order to reduce bundling inconsistency"
+    echo "\nOh no! You're not using our version of yarn, please move to version $desired_yarn_version in order to reduce bundling inconsistency"
```

Comment:
Did you do this on the Simorgh branch, I don't remember seeing it, the same script is in that repo too?

Reply:
Yep this was in the original Simorgh PR